include/eve/module/real/algorithm/function/regular/generic/reduce.hpp

```diff
@@ -54,26 +54,26 @@ namespace eve::detail
     return op(s, v);
   }
 
-  template<real_scalar_value T, typename N, typename Callable>
+  template<scalar_value T, typename N, typename Callable>
```

Comment:
Why this change ? I guess tests on tuple-like types ?

Reply:
Well reduce works on bundles. And somehow concepts restrictions here didn't work.

Reply:
Good